txtl/__init__.py

```diff
@@ -0,0 +1,28 @@
+# __init__.py - initialization of txtl toolbox
+# RMM, 11 Aug 2018
+
+# Core classes
+from .mixture import *
+from .mechanism import *
+from .component import *
+from .parameter import *
+
+# Core components
+from .extract import *
+from .dna import *
+
+# Additional functions
+from .sbmlutil import *
+from .bioscrape import runsim, plot
```

Comment:
This creates a dependency to __bioscrape__, should be moved closer to the interface.